src/Logger.ts

```diff
@@ -5,7 +5,7 @@ const WARN = 2;
 const ERROR = 1;
 const NONE = 0;
 
-let _logLevel = ERROR;
+let _logLevel = DEBUG;
```

Comment:
We may want to revert this

Reply:
Reverted it and moved the initialization of the logger to the top of setup